app/services/github_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class GithubService
+
+  def initialize(user)
+    @user = user
+  end
+
+  def get_repos
```

Comment:
Great way to set us up for the future feature implementation.  I like the way this method makes the service dynamic